app/models/org.rb

```diff
@@ -38,7 +38,7 @@ class Org < ActiveRecord::Base
                   :feedback_enabled, :feedback_email_subject, :feedback_email_msg
   ##
   # Validators
-  validates :contact_email, email: true, allow_nil: true
+#  validates :contact_email, email: true, allow_nil: true
```

Comment:
what's the case to remove this validator?

Reply:
I was asked to change the contact email so that it isn't required in the ticket. 

This `email: true` part though was causing it to be required even though we had `allow_nil: true`. I'll try switching the order so that its `allow_nil: true, email: true` perhaps that will work. We validate the email in the JS side so its not critical although it would be good to retain it here too